template/assets/style/app.styl

```diff
@@ -1,8 +1,21 @@
+// Import and define Vuetify color theme
+// https://vuetifyjs.com/en/style/colors
```

Comment:
Just realized that the theme does appear to be defined in the plugin, but there is no prompt(s) to enable or disable the lines. 

https://github.com/vuetifyjs/nuxt/blob/master/template/plugins/vuetify.js

```js
Vue.use(Vuetify{{#theme}}, { theme: {
  primary: '#9c27b0',
  accent: '#ce93d8',
  secondary: '#424242',
  info: '#0D47A1',
  warning: '#ffb300',
  error: '#B71C1C',
  success: '#2E7D32'
}}{{/theme}})
```